docs/applications/rubintv-dev/index.rst

```diff
@@ -0,0 +1,32 @@
+.. px-app:: rubintv-dev
+
+#################################################################
+rubintv-dev — Real-time display front end development application
+#################################################################
+
```

Comment:
Can we put a bit of text that flags this as a development app? I see there is something in the title, but a little more in the main body would be good.